services/minespace-web/src/components/Forms/projects/projectSummary/DocumentUpload.tsx

```diff
@@ -72,12 +62,25 @@ export const DocumentUpload: FC<DocumentUploadProps> = (props) => {
     ...IMAGE,
   };
 
+  useEffect(() => {
+    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "spatial_documents", []);
+    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "support_documents", []);
+  }, []);
+
+  useEffect(() => {
+    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "documents", [
```

Comment:
I don't think there's a field on the page anymore named documents, so I am wondering where this value is used

Reply:
The project summary form has a documents field that gets used in the back end.